bin/quest

```diff
@@ -65,6 +65,22 @@ module GLIWrapper
     output << "#{complete_count} of #{summary['example_count']} tasks."
   end
 
+  def wait_spinner
```

Comment:
You should consider using a library for this. I'm using the `spinner` gem in my own wrapper for vcmanager. https://github.com/randrews/spinner